tests/unit_tests/plugins/text/ascii_palettes/example_palettes/missing_docstring.txt

```diff
@@ -0,0 +1,259 @@
+# interface = ascii_palettes
+# family = rgb
+# name = missing_docstring
+47  6  66
+47  6  66
+47  6  66
+47  6  66
+47  6  66
+47  6  66
```

Comment:
I think these test files can probably be like 4 values long, the actual values shouldn't matter.

Reply:
Ok, I'll shorten these down quite a bit.

Reply:
Done!